### PR TITLE
#22660 - fixed

### PR DESCRIPTION
**Pull request by justinwilaby**

### CODE REVIEW

File: packages/app/client/src/ui/shell/mdi/tab.js [hunk 1]
```diff
@@ -105,11 +110,11 @@ class Tab extends React.Component {
...
     return (
-      <div className={ TAB_CSS + tabClassName } draggable
-        onDragOver={ this.onDragOver } onDragEnter={ this.onDragEnter } onDragStart={ this.onDragStart }
-        onDrop={ this.onDrop } onDragLeave={ this.onDragLeave } onDragEnd={ this.onDragEnd }>
+      <div className={TAB_CSS + tabClassName} draggable
+           onDragOver={ this.onDragOver } onDragEnter={ this.onDragEnter } onDragStart={ this.onDragStart }
+           onDrop={ this.onDrop } onDragLeave={ this.onDragLeave } onDragEnd={ this.onDragEnd }>
         <span className="editor-tab-icon"></span>
-        <span>{ this.props.title }</span>
+        <TruncateText className={truncateTextCss}>{ this.props.title }</TruncateText>
```
Comment (eanders-MS) [inline packages/app/client/src/ui/shell/mdi/tab.js]: nit: Whitespace after opening brace and before closing brace: `{ truncateTextCss }`

File: packages/app/client/src/ui/shell/mdi/tab.js [hunk 2]
```diff
@@ -105,11 +110,11 @@ class Tab extends React.Component {
     }
 
     return (
-      <div className={ TAB_CSS + tabClassName } draggable
-        onDragOver={ this.onDragOver } onDragEnter={ this.onDragEnter } onDragStart={ this.onDragStart }
-        onDrop={ this.onDrop } onDragLeave={ this.onDragLeave } onDragEnd={ this.onDragEnd }>
+      <div className={TAB_CSS + tabClassName} draggable
```
Comment (eanders-MS) [inline packages/app/client/src/ui/shell/mdi/tab.js]: Whitespace here as well.

File: packages/app/client/src/ui/shell/mdi/emulatorTab.js [hunk 3]
```diff
@@ -56,8 +54,8 @@ export class EmulatorTab extends React.Component {
 
   render() {
     return (
-      <Tab active={ this.props.active } title={ this.props.title } onCloseClick={ this.onCloseClick }
-        documentId={ this.props.documentId } dirty={ this.props.dirty } />
+      <Tab active={this.props.active} title={this.props.title} onCloseClick={this.onCloseClick}
```
Comment (eanders-MS) [inline packages/app/client/src/ui/shell/mdi/emulatorTab.js]: whitespace nit here too.

File: packages/app/client/src/ui/shell/mdi/tab.js (hunk 1 above)
Comment (justinwilaby) [reply packages/app/client/src/ui/shell/mdi/tab.js]: +1

File: packages/app/client/src/ui/shell/mdi/emulatorTab.js (hunk 3 above)
Comment (justinwilaby) [reply packages/app/client/src/ui/shell/mdi/emulatorTab.js]: Editor fail. I'll get that straightened out

**Pull request merged**